web-ng/src/app/components/observation-form/observation-form.component.css

```diff
@@ -42,3 +42,16 @@
 .mat-form-field-wrapper {
   margin-bottom: -1.34em;
 }
+/* The spacing between a field label and its options should be larger */
+.mat-label {
+  margin-bottom: 1em;
+}
+/* The spacing between radio buttons and checkboxes should be larger */
+.mat-radio-button,
+.mat-checkbox {
+  margin-right: 1em;
+}
+.asterix--after:after {
+  content: '*';
+  color: red;
```

Comment:
Please use Material themes for colors and fonts, in this case `mat-color($warn)` is probably the right one to use. Example in https://github.com/google/ground-platform/blob/44f31e2d115edd899f88579372d336964fa38c95/web-ng/src/styles.scss

Reply:
Done! Changed stylesheet to SCSS.